### DISCUSSION TITLE
火车读物 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#464] [General]
# 火车读物 - Miao Yu | 于淼

https://yufree.cn/cn/2024/05/27/train-reading/

**Comment by elizen** [comment]
可能是最近用的 AI 工具比较多，一看到首先，然后，之后，最后这样的开头，我就怀疑是 AI 写的。

**Reply by yufree** [reply]
有没有一种可能，我也是天天跟AI聊天被反向污染了。

**Reply by elizen** [reply]
其实还好，可能我比较敏感，没有 AI 的时候，也觉得这种风格，是受过某些训练养成的，比如体制内的公文写作，学术报告和论文什么的。我也写过几年程式化的东西，所以就敏感了。

**Reply by yufree** [reply]
想起一篇老文《反对党八股》，以后就看AI会不会写一篇《反对AI八股》的罪己诏了。
